types/components/http/Request.d.ts

```diff
@@ -160,13 +160,13 @@ export default class Request {
      * Returns path parameters from incoming request in Object form {key: value}
      * @returns {Record<string, unknown>}
      */
-    get path_parameters(): Object;
+    get path_parameters(): Record<string, unknown>;
```

Comment:
This should be Record<string, string> as path parameter keys/values can and will only ever be strings.

Reply:
fixed